Newsletters: add 89 (2020-03-18)

harding opened the pull request:
- [x] Services/Clients section @bitschmidty 
- [x] `LND #3821` @dongcarl 

Another short one this week.

adamjonas commented on _posts/en/newsletters/2020-03-18-newsletter.md:
```diff
@@ -0,0 +1,74 @@
+- [Bitcoin Core #16902][] changes consensus code to fix an inefficiency
+  in the parsing of `OP_IF` and related opcodes.  In legacy and segwit
+  v0 script, the inefficiency isn't believed to cause any significant
+  problems, but the proposal for [tapscript][topic tapscript] would
```
This sentence is a couple of ideas. Suggest breaking it up.

```suggestion
  problems. Still, the proposal for [tapscript][topic tapscript] would
```

dongcarl commented:
An attempt:

LND #3821 adds anchor commitments for LN channels and enables it by default if both participating nodes of a channel signal support. Anchor commitment transactions can be fee bumped unilaterally by either party, which is useful because commitment transactions might be broadcast a long time after they commit to their on-chain feerate. See the "Anchor outputs" topic for more details.

harding commented:
@dongcarl LGTM, thanks!  Added to the newsletter as written except that I removed the last sentence and placed its link in the first sentence (without otherwise editing that sentence).

Also ACK @bitschmidty's section.  Thanks!

jnewbery commented on _posts/en/newsletters/2020-03-18-newsletter.md:
```diff
@@ -0,0 +1,94 @@
+  merged this week.
+
+- [LND #3821][] adds [anchor commitments][topic anchor outputs] for
+  LN channels and enables it by default if both participating nodes of a
```
I'm not sure 'it' is correct here. It refers to "anchor commitments" which is plural, so suggests to me that we should use 'them'. Alternatively, "LND 3821 implements anchor commitments for enables this feature for new LN channels by default ..." or similar

jonatack replied:
Agree, was about to suggest s/it/this feature/

jnewbery commented:
Pushed a commit to link to the github version of the scaling book

jonatack commented on _posts/en/newsletters/2020-03-18-newsletter.md:
```diff
@@ -0,0 +1,94 @@
+generating a single onchain transaction, multiple payments will be
+[combined into a single transaction][scaling payment batching] once every 10 minutes.
+
+- **Bitstamp supports bech32:** Bitstamp users can now benefit from using native
```
Perhaps hoist "deposits and withdrawals" to the bitstamp title for clear quick comparison to the derebit title while skim reading.

jonatack commented on _posts/en/newsletters/2020-03-18-newsletter.md:
```diff
@@ -0,0 +1,94 @@
+[news85 ln stuck]: /en/newsletters/2020/02/19/#c-lightning-3500
+[coinbase batching blog]: https://blog.coinbase.com/coinbase-rolls-out-bitcoin-transaction-batching-5f6d09b8b045
+[bitstamp bech32 blog]: https://www.bitstamp.net/article/weve-added-support-bech32-bitcoin-addresses-bitsta/
+[deribit bech32 withdrawal tweet]: https://twitter.com/DeribitExchange/status/1234904442169851909
```
Tried to find a better link for this on deribit's blog but it seems to be down currently (which surely explains why the tweet is being used here instead).

bitschmidty replied:
There wasn’t anything on the blog when I looked (and it was up)

jonatack commented on _posts/en/newsletters/2020-03-18-newsletter.md:
```diff
@@ -0,0 +1,94 @@
+
+- [LND #3821][] adds [anchor commitments][topic anchor outputs] for
+  LN channels and enables it by default if both participating nodes of a
+  channel signal support. Anchor commitment transactions can be fee
```
Looked at the changeset to verify that anchor commitments are auto-enabled if both participants signal support"; comment and code seems to be here, lines 1114-1126: https://github.com/lightningnetwork/lnd/pull/3821/files#diff-691ca64693e704fd2e5080998f3e640eR1114

jonatack commented on _posts/en/newsletters/2020-03-18-newsletter.md:
```diff
@@ -0,0 +1,94 @@
+  commit to their on-chain feerate.
+
+- [LND #3963][] adds detailed [documentation][lnd op safety] about how
+  to use LND safely.
```
This looks like a PR of truly high value to users that might be worth a bit more pizzazz here. FWIW I like the name "lnd Operational Safety Guidelines".

jonatack commented on _posts/en/newsletters/2020-03-18-newsletter.md:
```diff
@@ -0,0 +1,94 @@
+- [LND #3963][] adds detailed [documentation][lnd op safety] about how
+  to use LND safely.
+
+- [Eclair #1319][] implements the same solution as described in
```
ISTM the PR title or a short description could be useful here, e.g. Eclair #1319 "Funder reserve for future fee increase" or descriptive equivalent, before writing that it is the same solution as described in Newsletter #85.

jonatack commented on _posts/en/newsletters/2020-03-18-newsletter.md:
```diff
@@ -0,0 +1,94 @@
+  inefficiency now reduces the number of changes that need to be made in the
+  proposed schnorr, taproot, and tapscript soft fork.  For more
+  information, see the Bitcoin Core PR Review Club [meeting notes][club
+  #16902] about this PR and a [related PR][Bitcoin Core #18002] also
```
Agree.

jonatack commented:
(verified the links so far as well)

jnewbery commented:
@jonatack 
> Question: what plans for the review club feature (and frequency)?

Plan is one per month, on the second week, with a summary of the most interesting discussion over the previous month's meetings.

That gives the following schedule:
- 1st week: no special section
- 2nd week: Bitcoin Core PR Review Club
- 3rd week: Changes to services and client software
- 4th week: Selected Q&A from Bitcoin StackExchange

Let me know if you want to write some of the Bitcoin Core PR Review Club sections.

jonatack commented:
> Plan is one per month, on the second week, with a summary of the most interesting discussion over the previous month's meetings.
> 
> Let me know if you want to write some of the Bitcoin Core PR Review Club sections.

Sure :+1:
